src/CMakeLists.txt

```diff
@@ -325,8 +325,15 @@ if(NOT TARGET RapidJSON::RapidJSON)
     INTERFACE_INCLUDE_DIRECTORIES "${CMAKE_CURRENT_SOURCE_DIR}/s3select/rapidjson/include")
 endif()
 
-option(WITH_FMT_HEADER_ONLY "use header-only version of fmt library" OFF)
-find_package(fmt 8.1.1...<10.0.0)
+option(WITH_FMT_HEADER_ONLY "use header-only version of fmt library" off)
```

Comment:
nit, i'd prefer to be more consistent if possible. can we use `OFF` here? if not, could you help me understand why `off` is better than `OFF`?

Reply:
Unintentional, fixed.